src/dev/windows/lv_windows_display.c

```diff
@@ -0,0 +1,173 @@
+/**
+ * @file lv_windows_display.c
+ *
+ */
+
+/*********************
+ *      INCLUDES
+ *********************/
+
+#include "lv_windows_display.h"
+#if LV_USE_WINDOWS
+
+#include "lv_windows_context.h"
+
+#include <process.h>
+
+/*********************
+ *      DEFINES
+ *********************/
+
+/**********************
+ *      TYPEDEFS
+ **********************/
+
+/**********************
+ *  STATIC PROTOTYPES
+ **********************/
+
+static unsigned int __stdcall lv_windows_display_thread_entrypoint(
+    void * parameter);
+
+/**********************
+ *  STATIC VARIABLES
+ **********************/
+
+/**********************
+ *      MACROS
+ **********************/
+
+/**********************
+ *   GLOBAL FUNCTIONS
+ **********************/
+
+lv_display_t * lv_windows_create_display(
+    const wchar_t * title,
+    int32_t hor_res,
+    int32_t ver_res,
+    int32_t zoom_level,
+    bool allow_dpi_override,
+    bool simulator_mode)
+{
+    lv_windows_create_display_data_t data;
+
+    lv_memzero(&data, sizeof(lv_windows_create_display_data_t));
+    data.title = title;
+    data.hor_res = hor_res;
+    data.ver_res = ver_res;
+    data.zoom_level = zoom_level;
+    data.allow_dpi_override = allow_dpi_override;
+    data.simulator_mode = simulator_mode;
+    data.mutex = CreateEventExW(NULL, NULL, 0, EVENT_ALL_ACCESS);
+    data.display = NULL;
+    if(!data.mutex) {
+        return NULL;
+    }
+
+    HANDLE thread = (HANDLE)_beginthreadex(
+                        NULL,
+                        0,
+                        lv_windows_display_thread_entrypoint,
+                        &data,
+                        0,
+                        NULL);
+    if(!thread) {
```

Comment:
LV_ASSERT_NULL(thread)

or

CloseHandle(data.mutex);

Note that exception handling needs to consider whether there is a resource leak. If dealing with resource leaks is more cumbersome, it is more recommended to use LV_ASSERT to directly assert processing to facilitate capturing the first scene.